src/reflux/NavigationStore.js

```diff
@@ -10,11 +10,12 @@ import PaymentDetails from 'src/containers/PaymentDetails';
 import ConfirmOrder from 'src/containers/ConfirmOrder';
 import Receipt from 'src/containers/Receipt';
 
-import Actions from './Actions';
-
 // Target Containers
 import ProductPage from 'src/containers/ProductPage';
 
+import Actions from './Actions';
```

Comment:
Is this used?

Reply:
Yes, you set it as in the constructor below -> this.listenables = Actions;
So the store can subscribe to the actions that are triggered